src/internal.h

```diff
@@ -466,7 +466,7 @@ void _dispatch_log(const char *msg, ...);
 		} \
 	} while (0)
 #else
-static inline void _dispatch_assert(long e, long line) {
+static inline void _dispatch_assert(long e, size_t line) {
```

Comment:
can you fix the style while at it, it's unlike the rest of dispatch, it should be:

```C
static inline void
_dispatch_assert(long e, size_t line)
{
```

thanks